### PR TITLE
WIP - Metadata changes

### PR DESCRIPTION
**Pull request by get-itips**
Mostly added ms.reviewer field

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SecurityCompliance/auditing-troubleshooting-scenarios.md [hunk 1]
```diff
@@ -186,4 +187,4 @@ b. In the **Parameters** field, the condition of the inbox rule is displayed. In
 
 c. The *MoveToFolder* parameter specifies the action for the inbox rule; in this example, messages received from admin@alpinehouse.onmicrosoft.com are moved to the folder named *AdminSearch*. Also see the [New-InboxRule](https://docs.microsoft.com/powershell/module/exchange/mailboxes/new-inboxrule) article for a complete list of parameters that can used to define the action of an inbox rule.
 
-d. The **UserId** field indicate the user who created the inbox rule specified in the **ObjectId** field. This user is also displayed in the **User** column on the search results page.
\ No newline at end of file
+d. The **UserId** field indicate the user who created the inbox rule specified in the **ObjectId** field. This user is also displayed in the **User** column on the search results page.
```
Comment (JohanFreelancer9) [inline SecurityCompliance/auditing-troubleshooting-scenarios.md]:
```suggestion
d. The **UserId** field indicates the user who created the inbox rule specified in the **ObjectId** field. This user is also displayed in the **User** column on the search results page.
```

File: SecurityCompliance/compliance-manager-release-notes.md [hunk 2]
```diff
@@ -88,4 +89,4 @@ The following sections cover known issues to be resolved in upcoming releases of
  
 ### Language support
 
-- All languages are not supported for all webpages. If your local language is unsupported, view in US English.
\ No newline at end of file
+- All languages are not supported for all webpages. If your local language is unsupported, view in US English.
```
Comment (JohanFreelancer9) [inline SecurityCompliance/compliance-manager-release-notes.md]:
```suggestion
- All languages are not supported for all web pages. If your local language is unsupported, view in US English.
```

File: SecurityCompliance/compliance-score-methodology.md [hunk 3]
```diff
@@ -76,4 +77,4 @@ Here's the basic workflow for a typical Action Item:
 
 3. The Assessor validates the control and examines the evidence and marks the control as assessed and the results of the assessment.
 
-Once all the Controls associated with an Assessment are assessed, the Assessment is complete.
\ No newline at end of file
+Once all the Controls associated with an Assessment are assessed, the Assessment is complete.
```
Comment (JohanFreelancer9) [inline SecurityCompliance/compliance-score-methodology.md]:
```suggestion
Once all the controls associated with an assessment are assessed, the assessment is complete.
```

File: SecurityCompliance/compliance20/attorney-privilege-detection.md [hunk 4]
```diff
@@ -126,4 +127,4 @@ For example, the following screenshots show two documents; the first one contain
 
 ![Document without any labels](../media/AeDTaggingPanelNegative.png)
 
-After you review a document to see if it contains privileged content, you can tag the document with the appropriate tag.
\ No newline at end of file
+After you review a document to see if it contains privileged content, you can tag the document with the appropriate tag.
```
Comment (JohanFreelancer9) [inline SecurityCompliance/compliance20/attorney-privilege-detection.md]:
```suggestion
After you have reviewed a document to see if it contains privileged content, you can tag the document with the appropriate tag.
```

File: SecurityCompliance/create-and-manage-inactive-mailboxes.md [hunk 5]
```diff
@@ -125,4 +126,4 @@ If another employee takes on the job responsibilities of a former employee, or i
...
\ No newline at end of file
+If you no longer need to retain the contents of an inactive mailbox, you can permanently delete the inactive mailbox by removing the hold or removing the Office 365 retention policy applied to the inactive mailbox. If the mailbox was deleted more than 30 days ago, the mailbox will be marked for permanent deletion after you remove the hold, and the mailbox will become non-recoverable. If the mailbox was deleted within the last 30 days, you can still recover the mailbox after removing the hold or retention policy. For step-by-step procedures for removing a hold or a Office 365 retention policy to permanently delete an inactive mailbox, see [Delete an inactive mailbox in Office 365](delete-an-inactive-mailbox.md).
```
Comment (JohanFreelancer9) [inline SecurityCompliance/create-and-manage-inactive-mailboxes.md]:
```suggestion
If you no longer need to retain the contents of an inactive mailbox, you can permanently delete the inactive mailbox by removing the hold or removing the Office 365 retention policy applied to the inactive mailbox. If the mailbox was deleted more than 30 days ago, the mailbox will be marked for permanent deletion after you remove the hold, and the mailbox will become non-recoverable. If the mailbox was deleted within the last 30 days, you can still recover the mailbox after removing the hold or retention policy. For step-by-step procedures to remove a hold or an Office 365 retention policy to permanently delete an inactive mailbox, see [Delete an inactive mailbox in Office 365](delete-an-inactive-mailbox.md).
```
```suggestion
If you no longer need to retain the contents of an inactive mailbox, you can permanently delete the inactive mailbox by removing the hold or removing the Office 365 retention policy applied to the inactive mailbox. If the mailbox was deleted more than 30 days ago, the mailbox will be marked for permanent deletion after you remove the hold, and the mailbox will become non-recoverable. If the mailbox was deleted within the last 30 days, you can still recover the mailbox after removing the hold or retention policy. For step-by-step procedures for removing a hold or a Office 365 retention policy to permanently delete an inactive mailbox, see [Delete an inactive mailbox in Office 365](delete-an-inactive-mailbox.md).
```

Comment (JohanFreelancer9) [inline SecurityCompliance/create-and-manage-inactive-mailboxes.md]:
```suggestion
If you no longer need to retain the contents of an inactive mailbox, you can permanently delete the inactive mailbox by removing the hold or removing the Office 365 retention policy applied to the inactive mailbox. If the mailbox was deleted more than 30 days ago, the mailbox will be marked for permanent deletion after you remove the hold, and the mailbox will become non-recoverable. If the mailbox was deleted within the last 30 days, you can still recover the mailbox after removing the hold or retention policy. For step-by-step procedures to removing a hold or an Office 365 retention policy to permanently delete an inactive mailbox, see [Delete an inactive mailbox in Office 365](delete-an-inactive-mailbox.md).
```

File: SecurityCompliance/compliance20/reviewing-data-in-review-set.md [hunk 6]
```diff
@@ -20,4 +21,4 @@ description: ""
 # Review case data in Advanced eDiscovery
 
 
-Coming soon
\ No newline at end of file
+Coming soon
```
Comment (JohanFreelancer9) [inline SecurityCompliance/compliance20/reviewing-data-in-review-set.md]:
```suggestion
Coming soon.
```

File: SecurityCompliance/data-encryption-in-odb-and-spo.md [hunk 7]
```diff
@@ -65,4 +66,4 @@ In other words, there are three different types of stores involved in per-file e
...
 
-Each of these three storage components—the blob store, the Content Database, and the Key Store—is physically separate. The information held in any one of the components is unusable on its own. This provides an unprecedented level of security. Without access to all three it is impossible to retrieve the keys to the chunks, decrypt the keys to make them usable, associate the keys with their corresponding chunks, decrypt any chunk, or reconstruct a document from its constituent chunks.
\ No newline at end of file
+Each of these three storage components—the blob store, the Content Database, and the Key Store—is physically separate. The information held in any one of the components is unusable on its own. This provides an unprecedented level of security. Without access to all three it is impossible to retrieve the keys to the chunks, decrypt the keys to make them usable, associate the keys with their corresponding chunks, decrypt any chunk, or reconstruct a document from its constituent chunks.
```
Comment (JohanFreelancer9) [inline SecurityCompliance/data-encryption-in-odb-and-spo.md]:
```suggestion
Each of these three storage components - the blob store, the content database, and the key store - is physically separate. The information held in any one of the components is unusable on its own. This provides an unprecedented level of security. Without access to all three it is impossible to retrieve the keys to the chunks, decrypt the keys to make them usable, associate the keys with their corresponding chunks, decrypt any chunk, or reconstruct a document from its constituent chunks.
```

File: SecurityCompliance/datainvestigations/search-for-data.md [hunk 8]
```diff
@@ -78,4 +79,4 @@ To add data to an an evidenced set, select a search on the **Searches** tab, and
...
\ No newline at end of file
+Adding data to an evidence set is a long-running process. The process includes collecting the items the original data source from Office 365 ( for example, from mailboxes and sites), copying them to the Azure storage location (this copying process is also called *ingestion*), and then re-indexing the items. You can either track the progress on the **Jobs** tab or on the **Searches** tab in the **Added data to evidence** column. After the processing of evidence processing is completed, you can go to the **Evidence** tab, click the evidence set, and then start your investigation by searching, reviewing, tagging, and exporting the relevant data as necessary.
```
Comment (JohanFreelancer9) [inline SecurityCompliance/datainvestigations/search-for-data.md]:
```suggestion
Adding data to an evidence set is a long-running process. The process includes collecting the items of the original data source from Office 365 ( for example, from mailboxes and sites), copying them to the Azure storage location (this copying process is also called *ingestion*), and then re-indexing the items. You can either track the progress on the **Jobs** tab or on the **Searches** tab in the **Added data to evidence** column. After the processing of evidence processing is completed, you can go to the **Evidence** tab, click the evidence set, and then start your investigation by searching, reviewing, tagging, and exporting the relevant data as necessary.
```

File: SecurityCompliance/datainvestigations/view-people-of-interest-activity.md [hunk 9]
```diff
@@ -96,4 +97,4 @@ You can export the results of an audit log search to a comma separated value (CS
 
 3. After you select an export option, a message is displayed at the bottom of the window that prompts you to open the CSV file, save it to the Downloads folder, or save it to a specific folder
 
-For more information about viewing, filtering, or exporting audit log search results, see [Search the audit log in the Office 365](../search-the-audit-log-in-security-and-compliance.md).
\ No newline at end of file
+For more information about viewing, filtering, or exporting audit log search results, see [Search the audit log in the Office 365](../search-the-audit-log-in-security-and-compliance.md).
```
Comment (JohanFreelancer9) [inline SecurityCompliance/datainvestigations/view-people-of-interest-activity.md]:
```suggestion
For more information about viewing, filtering, or exporting audit log search results, see [Search the audit log in the Security & Compliance Center](../search-the-audit-log-in-security-and-compliance.md).
```

File: SecurityCompliance/office-365-encryption-risks-and-protections.md [hunk 10]
```diff
@@ -83,4 +84,4 @@ Some risk scenarios and the currently available encryption technologies that mit
...
 | SMTP TLS with partner organization | Exchange Online | TLS 1.2 with AES 256 | The TLS certificate for Exchange Online (outlook.office.com) is a 2048-bit SHA256RSA certificate issued by Baltimore CyberTrust Root. <br> <br> The TLS root certificate for Exchange Online is a 2048-bit sha1RSA certificate issued by Baltimore CyberTrust Root. <br> <br> Be aware that for security reasons, our certificates do change from time to time. | Yes |
 
-**TLS certificates referenced in this table are for US datacenters; non-US datacenters also use 2048-bit SHA256RSA certificates.*
\ No newline at end of file
+**TLS certificates referenced in this table are for US datacenters; non-US datacenters also use 2048-bit SHA256RSA certificates.*
```
Comment (JohanFreelancer9) [inline SecurityCompliance/office-365-encryption-risks-and-protections.md]:
```suggestion
**TLS certificates referenced in this table are for US datacenters; non-US datacenters also use 2048-bit SHA256RSA certificates.**
```

**Pull request closed without merging**